kalliope/core/RestAPI/views/settings_views.py

```diff
@@ -165,100 +173,198 @@ def set_mute(self):
+                                                                  flag_to_find="recognizer_multiplier",
+                                                                  is_boolean=False)
+
+        SettingEditor.set_recognizer_multiplier(recognizer_multiplier=recognizer_multiplier)
+
+        data = {
+            "recognizer_multiplier": recognizer_multiplier
+        }
+        return jsonify(data), 200
+
+    @requires_auth
+    def get_recognizer_energy_ratio(self):
         """
-        Return the current energy_threshold value from settings
+        Return the current recognizer_energy_ratio value from settings
 
         Curl test
-        curl -i --user admin:secret  -X GET  http://127.0.0.1:5000/settings/energy_threshold
+        curl -i --user admin:secret  -X GET  http://127.0.0.1:5000/settings/recognizer_energy_ratio
         """
 
-        if self.settings.options.energy_threshold is not None:
+        if self.settings.options.recognizer_energy_ratio is not None:
             data = {
-                "energy_threshold": self.settings.options.energy_threshold
+                "recognizer_energy_ratio": self.settings.options.recognizer_energy_ratio
             }
             return jsonify(data), 200
 
         # if no Order instance
         data = {
-            "error": "energy_threshold status not defined"
+            "error": "recognizer_energy_ratio status not defined"
         }
         return jsonify(error=data), 400
 
     @requires_auth
-    def set_energy_threshold(self):
+    def set_recognizer_energy_ratio(self):
         """
-        Set the Kalliope Core energy_threshold value
+        Set the Kalliope Core recognizer_energy_ratio value
 
         Curl test:
         curl -i -H "Content-Type: application/json" --user admin:secret  -X POST \
-        -d '{"energy_threshold": "6666"}' http://127.0.0.1:5000/settings/energy_threshold
+        -d '{"energy_threshold": "6666"}' http://127.0.0.1:5000/settings/recognizer_energy_ratio
         """
 
-        if not request.get_json() or 'energy_threshold' not in request.get_json():
+        if not request.get_json() or 'recognizer_energy_ratio' not in request.get_json():
             data = {
-                "Error": "Wrong parameters, 'energy_threshold' not set"
+                "Error": "Wrong parameters, 'recognizer_energy_ratio' not set"
             }
             return jsonify(error=data), 400
 
-        # get energy_threshold if present
-        energy_threshold = utils.get_value_flag_from_request(http_request=request,
-                                                             flag_to_find="energy_threshold",
-                                                             is_boolean=False)
+        # get if present
+        recognizer_energy_ratio = utils.get_value_flag_from_request(http_request=request,
+                                                                    flag_to_find="recognizer_energy_ratio",
```

Comment:
Suggestion: because we copy / use keyword like "recognizer_energy_ratio" many times accross the codem I think it could be interesting to centralise the reference thus extract these keywords to dedicated constant class(es).

Reply:
Hi @LaMonF thanks for the review. 
I think I understand what you propose, but I'm not sure how to accomplish this in a proper way.

Reply:
@lamonf do you have a pointer to help here?

Reply:
Or maybe you have time to refactor it by yourself after accepting the PR?

Reply:
It is not related to this feature directly. it is more an overall suggestion to improve the code base quality. I will take care of it myself as suggested.